### ISSUE TITLE
Beta 11 is not reachable

### ISSUE THREAD

**Issue by bogacg** [angular/flex-layout#489]
I've changed  my `package.json` for beta 11 but getting this error:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @angular/flex-layout@2.0.0-beta.11
```

**Comment by arlowhite** [comment]
I see this version in npm: `'2.0.0-beta.10-4905443': '2017-11-05T15:18:57.776Z'`

Was there an issue with the npm release?

**Comment by snarum** [comment]
I'm having trouble with angular 5.0.0 and flex, and hoping beta-11 will solve this.
```
@angular/flex-layout@2.0.0-beta.10 requires a peer of @angular/core@0.0.0-NG but none was installed.
```

**Comment by nikonthethird** [comment]
@snarum what you're seeing is #479, there's more information there.

In the meantime, as long as beta 11 is not available on NPM, something that worked for me is to use the Github builds instead of NPM, by changing your `package.json`:

```json
"@angular/flex-layout": "git+https://github.com/angular/flex-layout-builds.git"
```

`npm install` still complains that `@angular/flex-layout@2.0.0-beta.11-b01c2d7 requires a peer of @angular/core@~4.4.4 but none is installed.` but everything works so far.

**Comment by arlowhite** [comment]
If the newest flex-layout depends on Angular5, I think it would be good to follow the material2 example and bump the version to `5.0.0-*`. But if it's backwards compatible I suppose it should stay at `2.0.0-*`

**Comment by bogacg** [comment]
In my app I see Flex-Layout bugs like layout not being rendered consistently across all browsers supposedly it supports but I don't see any version related error messages for Angular 5. Also some bugs like window resize renders one way correctly but not other, team is aware of that. If there are patches need to be applied for different browsers by developers, they need to be clearly put into documents. If these are to be fixed by the team, they should. A project under Angular umbrella and not seeing any move forward is troubling.

**Comment by adeptues** [comment]
this project is dead

**Comment by jocor87** [comment]
I am getting the error:

`npm WARN @angular/flex-layout@2.0.0-beta.10-4905443 requires a peer of @angular/common@~4.4.4 but none is installed. You must install peer
dependencies yourself.`

**Comment by kylecordes** [comment]
I'm thinking there must be some bigger problem than just "made a release and forget to `npm publish`". When the team is back in action (there's just people, and people have things come up to distract from projects!) I'm hoping to find out it's a not-too-big problem.

**Comment by ardatan** [comment]
So any life sign?

**Comment by majodi** [comment]
I'm following the install instructions and see that @angular/core@~4.4.4 is required. What are the implications if I just go ahead and ignore the warning? Or should I wait for an update of this package?

**Comment by rwisniewski1101** [comment]
I have been using this with the newest angular versions without issues beyond the warnings from npm.

**Comment by ThomasBurleson** [comment]
@bogacg - We plan on releasing Beta.12 today with:

* fixes to npm releases
* dep changes to Angular ~5.1.0 and RxJS 5.5.x (in alignment with @angular/material)

**Comment by ThomasBurleson** [comment]
Fixed with today's 2.0.0-beta.12 release. See SHA 62457a5972 and [CHANGELOG](https://github.com/angular/flex-layout/blob/master/CHANGELOG.md)
